app/views/find-help.nunjucks

```diff
@@ -4,7 +4,7 @@
 <h1 class="local-header--title--question">Find a pharmacy</h1>
 {% endblock %}
 
-{% block pageTitle %}Find a pharmacy{% if errorMessage %} - We can't find the postcode '{{req_location}}'{% endif %}{% endblock %}
+{% block pageTitle %}{% if errorMessage %}{% if req_location %}Find a pharmacy - We can't find the postcode '{{req_location | upper }}'{% else %}Find a pharmacy - Enter a town, city or postcode, or use your location{% endif %}{% else %}Find a pharmacy{% endif %}{% endblock %}
```

Comment:
There is quite a bit of logic in here, can it be extracted into a function please, in the same way `getDisplayLocation` or `hasNoCountries` has been done? It would make it easier to test and clean up the view.